workspaces/vault/.changeset/gorgeous-files-happen.md

```diff
@@ -0,0 +1,6 @@
+---
+'backend': patch
+'app': patch
+---
+
+Added an example app to the Vault workspace
```

Comment:
you can skip this changeset as `backend` and `app` won't be released

Reply:
Done!